rockcraft/oci.py

```diff
@@ -344,15 +344,15 @@ def to_oci_archive(self, tag: str, filename: str) -> None:
         src_path = self.path / f"{name}:{tag}"
         _copy_image(f"oci:{str(src_path)}", f"oci-archive:{filename}:{tag}")
 
-    def set_default_user(self, user: str) -> None:
-        """Set the default runtime user for the OCI image.
+    def set_default_userid(self, userid: int) -> None:
+        """Set the default runtime userid for the OCI image.
 
-        :param user: name of the default user (must already exist)
+        :param userid: userid of the default user (must already exist)
         """
         image_path = self.path / self.image_name
-        params = ["--clear=config.entrypoint", "--config.user", user]
+        params = ["--clear=config.entrypoint", "--config.user", str(userid)]
         _config_image(image_path, params)
-        emit.progress(f"Default user set to {user}")
+        emit.progress(f"Default user set to {userid}")
```

Comment:
it would be nice to print the username too, cause people won't know what those large UIDs map to. Something like
`Default user set to {userid} ({username})`

Reply:
Done